Misc/NEWS.d/next/Library/2025-04-30-19-32-18.gh-issue-132969.EagQ3G.rst

```diff
@@ -0,0 +1 @@
+Prevent the ProcessPoolExecutor executor thread from tying to adjust worker processes after shutdown is called and object state has already been reset. Calling shutdown with wait=False resets the state of the ProcessPoolExecutor while still leaving the executor thread running. A combination of conditions including a worker process having terminated abormally and shutdown(wait=False) resulted in an exception when the executor thread tried to replace dead workers, and additionally the process could also potentially hang.
```

Comment:
IMO your first sentence is explicit enough. I just suggest ta modify by adding `immediatly` as `after shutdown immediatly is called` part.
Any comments or suggestions are welcome from other reviewers.

Reply:
In the `shutdown`method, when wait is true, there is a `join` on the ``_ExecutorManagerThread thread. 
That why I propose to add  'immediatly'. It 's up to you.